e2e-polybft/property/property_test.go

```diff
@@ -35,6 +35,7 @@ func TestProperty_DifferentVotingPower(t *testing.T) {
 		}
 
 		cluster := framework.NewTestCluster(t, int(numNodes),
```

Comment:
Thinking out loud:
maybe it is better to create a separate function called `NewPropertyTestCluster`, which would set `IsPropertyTest` to `true` and wrap `NewTestCluster`. That way we would be more explicit that it is a cluster created for the property-based test. The way it is currently, it is easy to forget that `WithPropertyTestLogging` needs to be called.

Reply:
Done: https://github.com/0xPolygon/polygon-edge/pull/1371/commits/4819acf67a6fd96c1e52460fee14cc0453f70368